testing/test_debugging.py

```diff
@@ -21,11 +21,10 @@
 
 
 @pytest.fixture(autouse=True)
-def pdb_env(request):
+def pdb_env(request, monkeypatch: MonkeyPatch):
```

Comment:
I *suspect* the idea here was to hook into the `pytester`'s monkeypatch lifetime but haven't checked it. I suggest keeping the idea though just to be safe. The `getfixturevalue` would change to `"pytester"` and `pytester.monkeypatch` would change to `pytester._monkeypatch` (with underscore).